pkg/rpc/client.go

```diff
@@ -70,23 +72,22 @@ func (c Commitment) MarshalJSON() ([]byte, error) {
 }
 
 const (
-	// CommitmentMax represents the most recent block confirmed by the cluster super-majority
-	//as having reached maximum lockout.
-	CommitmentMax Commitment = "max"
-	// CommitmentRoot Most recent block having reached maximum lockout on this node.
-	CommitmentRoot Commitment = "root"
-	// CommitmentSingleGossip represents the most recent block that has been voted on
-	//by the cluster super-majority (optimistic confirmation).
-	CommitmentSingleGossip Commitment = "singleGossip"
-	// CommitmentRecent represents the nodes most recent block
-	CommitmentRecent Commitment = "recent"
+	// CommitmentFinalized level offers the highest level of certainty for a transaction on the Solana blockchain.
+	// A transaction is considered “Finalized” when it is included in a block that has been confirmed by a
+	// supermajority of the stake, and at least 31 additional confirmed blocks have been built on top of it.
+	CommitmentFinalized Commitment = "finalized"
```

Comment:
For the usage of the exporter, do we actually need to use finalized blocks? This is ~15 seconds behind the latest roughly. For monitoring, max is probably good enough.